txnkv/txnsnapshot/snapshot.go

```diff
@@ -635,12 +635,18 @@ func (s *KVSnapshot) get(ctx context.Context, bo *retry.Backoffer, k []byte) ([]
 	if len(matchStoreLabels) > 0 {
 		ops = append(ops, locate.WithMatchLabels(matchStoreLabels))
 	}
-	if req.ReplicaReadType.IsFollowerRead() && replicaAdjuster != nil {
-		op, readType := replicaAdjuster(1)
-		if op != nil {
-			ops = append(ops, op)
+	if req.ReplicaReadType.IsFollowerRead() {
```

Comment:
Also need to set `BusyThresholdMs` to 0 when `s.version == math.MaxUint64 && !req.ReplicaReadType.IsFollowerRead()`